apps/sms/style/attachment.css

```diff
@@ -32,6 +32,27 @@
   height: 12.7rem; /* font-size = 1.7rem + 1rem margin */
 }
 
+/* Internal height of the iframe that contains the attachment draft */
+.attachment-container.nopreview[data-attachment-type="vcard"] {
+  /* 40x40 px placeholder + filename below */
+  height: 6.7rem; /* font-size = 1.7rem + 1rem margin */
+}
+
+/* The internal height of the attachment itself */
+.attachment[data-attachment-type="vcard"] {
+  height: 100%;
+}
+
+/* Allow to display the filename properly */
+#messages-input > [data-attachment-type="vcard"] {
+  width: 20rem;
+}
+
+/* To set the proper active state in attachment drafts */
```

Comment:
Since we don't need the active state for vcard anymore, do we still need to keep it?